weaviate/gql/filter.py

```diff
@@ -53,6 +54,16 @@
 ]
 
 
+class MediaType(Enum):
+    TEXT = "text"
```

Comment:
Also, is this ENUM value ever used in the code? I see that `NearText` still inherits from `Filter` rather than `NearMedia`. Is `MediaType.TEXT` reserved for filtering on `.txt` files?